dbt/models/default/default.vw_pin_status.sql

```diff
@@ -0,0 +1,153 @@
+    LEFT JOIN {{ source('iasworld', 'comdat') }} AS com
+        ON par.parid = com.parid
+        AND par.taxyr = com.taxyr
+        AND com.cur = 'Y'
+        AND com.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'dweldat') }} AS dwel
+        ON par.parid = dwel.parid
+        AND par.taxyr = dwel.taxyr
+        AND dwel.cur = 'Y'
+        AND dwel.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'oby') }} AS oby
+        ON par.parid = oby.parid
+        AND par.taxyr = oby.taxyr
+        AND oby.cur = 'Y'
+        AND oby.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'land') }} AS land
+        ON par.parid = land.parid
+        AND par.taxyr = land.taxyr
+        AND land.cur = 'Y'
+        AND land.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'aprval') }} AS aprval
+        ON par.parid = aprval.parid
+        AND par.taxyr = aprval.taxyr
+        AND aprval.cur = 'Y'
+        AND aprval.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'exadmn') }} AS admn
+        ON par.parid = admn.parid
+        AND par.taxyr = admn.taxyr
+        AND admn.cur = 'Y'
+        AND admn.deactivat IS NULL
+    WHERE par.cur = 'Y'
+        AND par.deactivat IS NULL
+    GROUP BY par.parid, par.taxyr
+),
+
+-- These CTEs make it easier to work with CDUs from oby, comdat, pardat since
+-- those tables aren't unique by parid and taxyr.
+dwel_cdu AS (
+    SELECT
+        dwel.parid,
+        dwel.taxyr,
+        ARRAY_JOIN(
+            ARRAY_SORT(ARRAY_DISTINCT(ARRAY_AGG(dwel.cdu))), ', '
+        ) AS cdu
+    FROM {{ source('iasworld', 'dweldat') }} AS dwel
+    WHERE dwel.cdu IS NOT NULL
+        AND dwel.cur = 'Y'
+        AND dwel.deactivat IS NULL
+    GROUP BY dwel.parid, dwel.taxyr
+),
+
+com_cdu AS (
+    SELECT
+        com.parid,
+        com.taxyr,
+        ARRAY_JOIN(
+            ARRAY_SORT(ARRAY_DISTINCT(ARRAY_AGG(com.user16))), ', '
+        ) AS cdu
+    FROM {{ source('iasworld', 'comdat') }} AS com
+    WHERE com.user16 IS NOT NULL
+        AND com.cur = 'Y'
+        AND com.deactivat IS NULL
+    GROUP BY com.parid, com.taxyr
+),
+
+oby_cdu AS (
+    SELECT
+        oby.parid,
+        oby.taxyr,
+        ARRAY_JOIN(
+            ARRAY_SORT(ARRAY_DISTINCT(ARRAY_AGG(oby.user16))), ', '
+        ) AS cdu
+    FROM {{ source('iasworld', 'oby') }} AS oby
+    WHERE oby.user16 IS NOT NULL
+        AND oby.cur = 'Y'
+        AND oby.deactivat IS NULL
+    GROUP BY oby.parid, oby.taxyr
+)
+
+SELECT
+    pdat.parid AS pin,
+    pdat.class,
+    pdat.taxyr AS year,
+    colo.is_corner_lot,
+    ahsap.is_ahsap,
+    COALESCE(vpe.pin IS NOT NULL, FALSE) AS is_exempt,
+    COALESCE(pin.tax_bill_total = 0, FALSE) AS is_zero_bill,
+    vpcc.is_parking_space,
+    vpcc.parking_space_flag_reason,
+    vpcc.is_common_area,
+    SUBSTR(pdat.parid, 11, 1) = '8' AS is_leasehold,
+    ptst.test_type AS weirdness,
+    oby.cdu AS oby_cdu,
+    cdat.cdu AS com_cdu,
+    ddat.cdu AS dwel_cdu,
+    pdat.note2 AS note
```

Comment:
**suggestions (blocking):** A few more things to add here:

- Railroad PIN
- "Filler" PIN (ends in 999 or has 999 in the PIN)
- Mixed use (commercial and res in one PIN)
- Has Chicago ADU

Reply:
I'll add class = '999' as well

Reply:
Chicago ADU might be something we want to add in another PR. It might end up being a whole ETL in itself.

For the mixed use indicator I've commented the logic and [attached output](https://github.com/user-attachments/files/17016850/mixed.use.indicator.csv from the CTE before it's aggregated so you can check to see if I understood the spirit of the column.